doc/sed_readme.sh

```diff
@@ -4,10 +4,10 @@ dir=$(cd "$(dirname $0)/.." && pwd)
 
 if [ -e $dir/README.md ]
 then
-	sed -e 's/\[!\[Build Status\](https:\/\/travis-ci.org\/outscale\/packetgraph.svg)\](https:\/\/travis-ci.org\/outscale\/packetgraph)/\\htmlonly \<a href=\"https:\/\/travis-ci.org\/outscale\/packetgraph\"\> \<img src=\"https:\/\/travis-ci.org\/outscale\/packetgraph.svg\"\> \<\/a\>\\endhtmlonly/g' $dir/README.md > $dir/doc/README2.md
-	sed -i 's/#betterfly/\\#betterfly/g' $dir/doc/README2.md
-	exit 0
+    sed -e 's/\[!\[Build Status\](https:\/\/travis-ci.org\/outscale\/packetgraph.svg)\](https:\/\/travis-ci.org\/outscale\/packetgraph) \[!\[packetgraph documentation\](doc\/packetgraph_doc.png)\](https:\/\/outscale.github.io\/packetgraph\/doc\/master)/\\htmlonly\<a href=\"https:\/\/travis-ci.org\/outscale\/packetgraph\"\> \<img src=\"https:\/\/travis-ci.org\/outscale\/packetgraph.svg\"\> \<\/a\>\\endhtmlonly/g' $dir/README.md > $dir/doc/README2.md
+    sed -i 's/#betterfly/\\#betterfly/g' $dir/doc/README2.md
+    exit 0
 else
-	echo "missing $dir/README.md file"
-	exit 1
+    echo "missing $dir/README.md file"
+    exit 1
```

Comment:
Shouldn't it be merge with `doc: add script and update Doxyfile for documentation`?